components/ws-manager-mk2/controllers/metrics.go

```diff
@@ -228,6 +228,8 @@ func (m *controllerMetrics) countWorkspaceStop(log *logr.Logger, ws *workspacev1
 	class := ws.Spec.Class
 	tpe := string(ws.Spec.Type)
 
+	log.Info("workspace stop reason", "type", tpe, "class", class, "reason", reason)
```

Comment:
We used to make it an object, and may need to convert into TrustedValue to avoid being redacted

Reply:
It is not redacted, please refer to the test link under How To Test